plugins/services/src/js/reducers/JSONMultiContainerReducers.js

```diff
@@ -29,14 +29,15 @@ module.exports = {
       joinedPath === "scheduling" ? Util.deepCopy(value) : state;
 
     const constraintsState =
-      state != null && state.placement != null
-        ? state.placement.constraints
-        : [];
+      scheduling.placement != null ? scheduling.placement.constraints : [];
```

Comment:
I’m not 100% sure about changing this line. I did it to be consistent with the line below which was required for the fix.